pipeline/00-ingest.R

```diff
@@ -55,6 +55,7 @@ training_data <- dbGetQuery(
       BETWEEN CAST({params$input$min_sale_year} AS int) -
         {params$input$n_years_prior}
       AND CAST({params$input$max_sale_year} AS int)
+  AND sale.deed_type IN ('01', '02', '05')
```

Comment:
There's a single deed that slips through all the other filters with the `deed_type` equal to `X`, this is to drop it.

Reply:
Could you mention this in a data architecture issue?